muscle3/muscle_manager.py

```diff
@@ -33,6 +33,25 @@
             ' a directory with the name run_<model>_<date>-<time> will be'
             ' created.')
         )
+@click.option(
+        '--location-file', nargs=1, type=click.Path(
+            exists=False, file_okay=True, dir_okay=True, readable=False,
+            allow_dash=True),
+        help=(
+            'File to write the network location of the manager to if'
+            ' --start-all is not specified. If a relative path is given,'
+            ' then it will be resolved relative to the directory in which'
+            ' the manager was started.'
+            ''
```

Comment:
These newlines are not preserved in the output of `muscle_manager --help` , not sure if that is your intention?

Reply:
Sort of, I was mainly too lazy to look up how to do it. Turns out there is a way though, and you're right, so I fixed it.